DARE-1.1.md

```diff
@@ -257,29 +257,31 @@ generator (CRSPRNG) should be used to generate `R`. Most operating systems
 already provide such a CRSPRNG - like `/dev/urandom` under linux. Furthermore
 many programming languages provide an easy-to-use interface to access a
 platform-specific CRSPRNG.
+
 Often programming languages provide an additional random number generator
 (RNG) - often in the context of math package/library - which is not designed
 to be cryptographically-secure and therefore must not be used.
 
 ### 5.2 Side channels
 
 The decrypted payload of a package should never be processed before the
-authentication  tag of the package is not verified. Any decrypted but not
-yet verified data could be modified by an adversary. Additionally the
-authentication tag should be verified using a constant-time comparison
-to prevent a timing side channel. This is ideally done by the AEAD cipher
+authentication tag of the package is verified. Any such decrypted but as yet
+un-verified data could have be modified by an adversary. Additionally, the
+authentication tag should be verified using a constant-time comparison to
+prevent a timing side channel. This is ideally done by the AEAD cipher
 implementation itself.
 
 ## 6. Secret key
 
 The secret key should be unique per encrypted data stream. Since it is 256 bits
-long the probability of choosing a specific key more than once is negligible
-(<code>< ~2<sup>-128</sup></code>) if the secret key is chosen uniformly at
-random. There are multiple ways to choose a secret key:
+long the probability of choosing a previously chosen key again is negligible
```

Comment:
`Since it is 256 bits long, the probability of choosing a previously used key again is negligible.`